modules/custom/openy_campaign/src/Form/MemberRegistrationPortalForm.php

```diff
@@ -107,6 +107,8 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
+    // The block is used by site admins.
+    // We shouldn't cache it.
```

Comment:
we have user cache tags, or user context. https://www.drupal.org/docs/8/api/cache-api/cache-contexts
Let's work on this after release.